test/integration/scipy_test.py

```diff
@@ -0,0 +1,138 @@
+# skip-if: True
+# This test will take a long time(51mins).
+# 3100.61s user 490.14s system 118% cpu 50:30.63 total
+# So please test it manually.
+
+import os
+import sys
+import subprocess
+sys.path.append(os.path.dirname(__file__) + "/../lib")
+from test_helper import run_test
+
+"""
+Using this test file.
+
+SciPy C source code files generated individually by Cython, it may contains
+implicit declaration of functions when compiling. Pyston set the flag
+"-Werror=implicit-function-declaration". It will trigger errors like:
+"error: implicit declaration of function 'zcopy_' " when building SciPy.
+This option was disabled in CPython. But enabled in Pyston. So if you want build
+and test SciPy in Pyston, please revert the second change in
+https://github.com/dropbox/pyston/commit/949dfeb1f675ef05ddfd04333e7e6c0f9887860c
```

Comment:
For `implicit declaration of function` error, here is an example:
```
scipy/sparse/linalg/dsolve/SuperLU/SRC/clacon.c:157:5: error: implicit declaration of function ‘ccopy_’ [-Werror=implicit-function-declaration]
     ccopy_(n, x, &c__1, v, &c__1);
```

I took a look at SciPy source code in `scipy/sparse/linalg/dsolve/SuperLU`. It declared the function signature by `extern`. But the function implementation is in fortran code. If my understanding is correct. There has no other way except revert the change in that commit. Let the behavior more like CPython. Thoughts?